README.md

```diff
@@ -44,42 +38,45 @@ To use Failsafe Actuator, add the following dependency to your project:
 
 **Gradle:**
 ```groovy
-compile("org.zalando:failsafe-actuator:0.4.1")
+compile("org.zalando:failsafe-actuator:${FAILSAFE-ACTUATOR-VERSION}")
 ```
 
 **Maven:**
 ```xml
 <dependency>
     <groupId>org.zalando</groupId>
     <artifactId>failsafe-actuator</artifactId>
-    <version>FAILSAFE-ACTUATOR-VERSION</version>
+    <version>${FAILSAFE-ACTUATOR-VERSION}</version>
 </dependency>
 ```
 
-Then autowire the `CircuitBreaker` by using:
+Create your `CircuitBreaker` by defining them as a `Bean`.
 
 ```java
-@Autowired
-@FailsafeBreaker("WhatABreak")
-CircuitBreaker breaker;
+@Configuration
+public class CircuitBreakerConfiguration {
+  @Bean("WhatABreak")
+  public CircuitBreaker createBreaker() {
+    return new CircuitBreaker();
+  }
+}
 ```
 
-This will inject a new instance of a circuit breaker to your bean and register it for monitoring. Example:
+You can use and configure the created `CircuitBreaker` by autowiring it in the class where it should be used.
+
 
 ```java
 @Component
 public class MyBean {
-    
         @Autowired
-        @FailsafeBreaker(value = "WhatABreak")
+        @Qualifier(value = "WhatABreak")
         private CircuitBreaker breaker;
-        
 }
 ```
 
-The [endpoint](http://docs.spring.io/spring-boot/docs/current/reference/html/production-ready-endpoints.html) is reachable via _**http://${yourAddress}/failsafe**_.
+That's it. By calling the [endpoint](http://docs.spring.io/spring-boot/docs/current/reference/html/production-ready-endpoints.html) via _**http://${yourAddress}/failsafe**_.
```

Comment:
By calling the endpoint you ....? It feels like the rest of the sentence is missing.

Reply:
Fixed